src/app/components/site/Homepage.tsx

```diff
@@ -108,20 +115,32 @@ export const Homepage = () => {
                 <hr/>
             </Container>}
 
-            {!user?.loggedIn && <section id="why-sign-up" className="row sign-up-tabs">
+            {!user?.loggedIn && <><section id="why-sign-up" className="row sign-up-tabs">
                 <Container>
                     <Col className="pb-5 pt-4 pattern-04">
                         <h2 className="text-center mb-5">Why sign up?</h2>
                         <WhySignUpTabs/>
                     </Col>
                 </Container>
-            </section>}
+            </section>
+
+            <section data-testid="promo-content" id="promo-content" className="row bg-primary pattern-05">
+                <ShowLoading
+                until={promoItem}
+                thenRender={()=><Container>
+                    <Col className="py-5">
+                        {promoItem && <PromoContent item={promoItem} />}
+                    </Col>
+                </Container>}
+                />
+                </section></>
```

Comment:
The tabulation here is not right and it makes it harder to read.

Reply:
This is the usual issue with Prettier not being in use; I can easily format the file to make it look better but that'll make a lot more changes and make the PR harder to spot what actually changed other than indentation.

Reply:
Yeah... :-( In the meanwhile you can just ensure that the lines you modify have the right indentation. There may be a way to configure your IDE to do this automatically without having to modify the whole file.